config/deploy.rb

```diff
@@ -0,0 +1,50 @@
+# config valid only for current version of Capistrano
+lock '3.4.0'
+
+set :application, 'MythicApps-onRails'
+set :repo_url, 'https://github.com/MythicApps/MythicApps-onRails'
+
+# Default branch is :master
+# ask :branch, `git rev-parse --abbrev-ref HEAD`.chomp
+set :branch, 'atomaka/bugfix/duplicate-gems'
```

Comment:
Needs to be removed so default branch will be master.  The duplicate gems in the Gemfile currently causes console warnings that stop capistrano install so this is needed for testing.